lib/common_test/src/ct_ftp.erl

```diff
@@ -38,7 +36,11 @@ FTP client module (based on the `ftp` application).
 
 -define(DEFAULT_PORT,21).
 
--doc "For `target_name`, see module `m:ct`.".
+-doc """
+Reference to opened FTP connection associated to either a `handle` or `target_name`.
+
+For `target_name`, see module `m:ct`.
```

Comment:
I think we can delete this as the type itself contains a link now to where more information is available.

Reply:
Will fix, thank you :)

Reply:
Fixed this case, as well as other similar cases.